compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SymbolEnter.java

```diff
@@ -1152,11 +1154,15 @@ public void initPredeclaredModules(Map<Name, BPackageSymbol> predeclaredModules,
                                        List<BLangCompilationUnit> compUnits, SymbolEnv env) {
         SymbolEnv prevEnv = this.env;
         this.env = env;
-        for (Name alias : predeclaredModules.keySet()) {
+        Name alias;
+        BPackageSymbol value;
+        for (Map.Entry<Name, BPackageSymbol> e : predeclaredModules.entrySet()) {
+            alias = e.getKey();
             int index = 0;
             ScopeEntry entry = this.env.scope.lookup(alias);
+            value = e.getValue();
```

Comment:
We can probably move variable declarations inside the loop I guess since we don't use them outside of the loop.
Shall we rename `value` to a different name like `packageSymbol`

Reply:
If we add variable declaration to inside the loop, Program needs to initialize the variable again and again. Isn't it a. added overhead?

Reply:
+1 to rename the variable

Reply:
You anyway initialize it in every loop right. Assuming compiler does not optimize this at all, there will be small imapct which will be small even to measure.
But as this is a suggestion you can keep as it is too :).

Reply:
Added the suggestions. 0dc515a2fa1f85d32b05c4328836af9dc8e09bf8